### PR TITLE
Add contextual tuples to Expand API

### PR DESCRIPTION
**Pull request by sujitha-av**
## Description

Based on [this](https://github.com/openfga/openfga/issues/951) issue in the openfga repo, it was required to add contextual tuples support to the Expand API. And @miparnisari had broken down the steps for the same. This PR does the first step mentioned, to add contextual tuples to the Expand API.

The technical documentation has been skipped.

## References
[Reference issue](https://github.com/openfga/openfga/issues/951)

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: openfga/v1/openfga_service.proto [hunk 1]
```diff
@@ -1183,7 +1183,9 @@ message ExpandRequest {
     (google.api.field_behavior) = REQUIRED
   ];
 
-  string authorization_model_id = 3 [
+  openfga.v1.ContextualTupleKeys contextual_tuples = 3 [json_name = "contextual_tuples"];
```
Comment (miparnisari) [inline openfga/v1/openfga_service.proto:1186]: Please do not alter the number of the existing fields (see https://github.com/openfga/api/pull/202/files)

Please put the new field at the end of this list with value `5`

**Review by miparnisari — CHANGES_REQUESTED** [review]
After applying the suggestion I made, please run `make` before commiting. Thanks!

File: openfga/v1/openfga_service.proto (hunk 1 above)
Comment (sujitha-av) [reply openfga/v1/openfga_service.proto]: Noted, the changes have been made. 👍

**Review by miparnisari — DISMISSED** [review]

**Comment by miparnisari** [comment]
@sujitha-av looks like there is a conflict with main. Could you please `git merge origin/main`?

**Comment by rhamzeh** [comment]
@miparnisari please make sure not to merge this until the relevant openfga/openfga PR is ready to be merged, otherwise autogenerated SDKs & Docs would be wrong

**Review by miparnisari — APPROVED** [review]

**Pull request merged**